### ISSUE TITLE
After make, ./eth throws error

### ISSUE THREAD

**Issue by conradjohnson** [ethereum/cpp-ethereum#1896]
Commands: /eth -G -M, ./eth -G

error:
terminate called after throwing an instance of 'cl::Error'
  what():  clGetPlatformIDs
Aborted (core dumped)

System - Ubuntu 14.04 (latest updates installed), AMD CPU, AMD r9 290 GPU, with AMD drivers installed (.deb files from AMD support).  

Followed installation instructions with the newly added 'opencl-headers', downloaded latest from github, cmake with 'DETHASHCL=1 ..' and a successful make.  

Can run ./eth without GPU mining successfully.

**Comment by conradjohnson** [comment]
Have tried a number of different Ubuntu versions (14.04.x) and AMD driver combinations, but I always get stuck here: 
terminate called after throwing an instance of 'cl::Error'
  what():  clGetPlatformIDs
Aborted (core dumped)

**Comment by debris** [comment]
Can you check if instructions from #1897 help? 

TL;DR

```bash
apt-get install nvidia-modprobe clinfo
```

If yes, we will add this to our wiki and build script.
